### PR TITLE
Debug subscribers

### PR DESCRIPTION
**Pull request by vangheem**
@bloodbare thoughts on this approach?

You would be responsible for the logging configuration to do something with this data.

### CODE REVIEW

**Comment by bloodbare** [comment]
Whould be great some documentation and instructions how to use it (besides changelog)

File: guillotina/component/globalregistry.py [hunk 1]
```diff
@@ -46,13 +51,73 @@ def subscribers(self, objects, provided):
...
+
+
 class GuillotinaAdapterRegistry(AdapterRegistry):
     """
     Customized adapter registry for async
     """
 
     _delegated = AdapterRegistry._delegated + ("asubscribers",)  # type: ignore
-    LookupClass = GuillotinaAdapterLookup
+    if os.environ.get("DEBUG_SUBSCRIBERS") in ("1", "true", "t") or True:
```
Comment (masipcat) [inline guillotina/component/globalregistry.py]: Is the `or True` correct?

File: guillotina/component/globalregistry.py [hunk 2]
```diff
@@ -46,13 +51,73 @@ def subscribers(self, objects, provided):
...
+        except RequestNotFound:
+            request = None
+        try:
+            url = request.url.human_repr()
+        except AttributeError:
+            # older version of aiohttp
+            url = ""
+        info = {
+            "url": url,
+            "account": getattr(task_vars.container.get(), "id", None),
```
Comment (bloodbare) [inline guillotina/component/globalregistry.py]: should be container

File: guillotina/component/globalregistry.py (hunk 1 above)
Comment (vangheem) [reply guillotina/component/globalregistry.py]: Nope! Fixed :)

File: guillotina/component/globalregistry.py [hunk 3]
```diff
@@ -46,13 +51,73 @@ def subscribers(self, objects, provided):
...
+
+
 class GuillotinaAdapterRegistry(AdapterRegistry):
     """
     Customized adapter registry for async
     """
 
     _delegated = AdapterRegistry._delegated + ("asubscribers",)  # type: ignore
-    LookupClass = GuillotinaAdapterLookup
+    if os.environ.get("GDEBUG_SUBSCRIBERS") in ("1", "true", "t"):
```
Comment (bloodbare) [inline guillotina/component/globalregistry.py]: maybe better:

`os.environ.get("GDEBUG_SUBSCRIBERS", "").lower() in ("1", "true", "t")`

Comment (vangheem) [reply guillotina/component/globalregistry.py]: thanks, updated!

**Pull request merged**